tests/components/PixLink.test.js

```diff
@@ -0,0 +1,246 @@
+import {
+  removeHostIfCurrentSite,
+  getPathLocale,
+  isCurrentLocaleURL,
+} from '~/components/PixLink'
+import { SITES_PRISMIC_TAGS } from '~/services/available-sites'
+
+describe('Component: PixLink', () => {
+  describe('removeHostIfCurrentSite()', () => {
```

Comment:
```suggestion
  describe('#removeHostIfCurrentSite', () => {
```

Pour matcher avec le nommage utilisé chez Pix